backend/pick-git/src/main/java/com/woowacourse/pickgit/post/presentation/PostController.java

```diff
@@ -246,24 +237,18 @@ private PostDeleteRequestDto createPostDeleteRequestDto(AppUser user, Long postI
         @Authenticated AppUser appUser,
         @PathVariable Long postId
     ) {
-        AuthUserForPostRequestDto authUserRequestDto = createAuthUserForPostRequestDto(appUser);
+        AuthUserForPostRequestDto authUserRequestDto =
+            new AuthUserForPostRequestDto(appUser);
 
         List<LikeUsersResponseDto> likeUsersResponseDtos = postService
             .likeUsers(authUserRequestDto, postId);
```

Comment:
```suggestion
        List<LikeUsersResponseDto> likeUsersResponseDtos = 
            postService.likeUsers(authUserRequestDto, postId);
```

지금까지 리팩토한 내용을 생각하면, 이렇게 개행해야 할 것 같아요 !